SamplePlugin/SamplePlugin.cs

```diff
@@ -7,9 +7,9 @@
 
 namespace SamplePlugin
 {
-    [Plugin(Name = "Sample Plugin", Version = "1.0",
-            Authors = "Obsidian Team", Description = "My sample plugin.",
-            ProjectUrl = "https://github.com/Naamloos/Obsidian")]
```

Comment:
These values were chosen to be explicit for the sample plugin. If possible, please keep these explicit.

Reply:
In the next commit, they will be explicit.